src/integrationTest/java/com/mongodb/hibernate/query/AbstractQueryIntegrationTests.java

```diff
@@ -36,17 +38,17 @@
 
 @SessionFactory(exportSchema = false)
 @ExtendWith(MongoExtension.class)
-abstract class AbstractSelectionQueryIntegrationTests implements SessionFactoryScopeAware, ServiceRegistryScopeAware {
+public abstract class AbstractQueryIntegrationTests implements SessionFactoryScopeAware, ServiceRegistryScopeAware {
```

Comment:
this class is reusable not only for `select` query but also for `mutate` query, so it is renamed and moved to more general location. The `Book` class was changed accordingly to go together with it hand in hand.